### PR TITLE
Fix SpinnerSliderPreference Class, Update Mouse Sensitivity Menu

### PR DESCRIPTION
**Pull request by r3tk0n**
Changed SpinnerSliderPreference.qml to general-case UI element as originally intended, updated PreferencesDialog.cpp's Avatar Scale slider to hard-coded bounds [0.25, 4.0]. Though the Avatar Scale slider will be removed with the inclusion of the Avatar App, so that is the least important part of this PR.

### CODE REVIEW

File: interface/src/ui/PreferencesDialog.cpp [hunk 1]
```diff
@@ -286,6 +288,7 @@ void setupPreferences() {
         preferences->addPreference(new CheckPreference(MOVEMENT, "Flying & jumping", getter, setter));
     }
     {
+
```
Comment (danteruiz) [inline interface/src/ui/PreferencesDialog.cpp]: remove this new line.

Comment (r3tk0n) [reply interface/src/ui/PreferencesDialog.cpp]: Done ^^

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11032/HighFidelity-Beta-PR13511-39f491d.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11032/HighFidelity-Beta-PR13511-39f491d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11032/HighFidelity-Beta-Interface-PR13511-39f491d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11032/HighFidelity-Beta-PR13511-39f491d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11032/HighFidelity-Beta-Interface-PR13511-39f491d.exe

**Comment by qliejahner** [comment]
Currently testing

**Comment by qliejahner** [comment]
PASS @conklin94122 

The spinners and sliders work

**Pull request merged**